web/src/containers/admin/MyAccount.test.js

```diff
@@ -87,10 +87,7 @@ describe('my account page', () => {
       position: 'Table Cleaner'
     };
 
-    // sinon.stub().rejects() wraps strings in Error objects, but the edit
-    // account action rejects a plain string, so return a rejected promise
-    // eslint-disable-next-line prefer-promise-reject-errors
-    const editAccount = sinon.stub().returns(Promise.reject('error message'));
+    const editAccount = sinon.spy();
```

Comment:
For a hot minute, these admin forms relied on Redux actions returning promises.  That wasn't a good design and we changed it, but this test was still built around returning promises.  The promise was rejected, and the `MyAccount` component no longer handled it.  Easy fix, though - just take out the promise!